src/components/backupWallet/ExportWalletData.js

```diff
@@ -62,45 +64,72 @@ const ExportWalletData = ({ navigation, styles, theme }: ExportWalletProps) => {
   return (
     <Wrapper style={styles.wrapper}>
       <NavBar title="EXPORT MY WALLET" goBack={handleGoHome} />
-      <Section grow>
-        <View style={styles.containerForBoxes}>
-          <BorderedBox
-            styles={styles}
-            theme={theme}
-            title="My Private Key"
-            content={fullPrivateKey}
-            truncateContent
-            imageSize={50}
-            imageSource={avatar}
-            image={!avatar && unknownProfile}
-            copyButtonText="Copy Key"
-            showCopyIcon={false}
-          />
-          <BorderedBox
-            styles={styles}
-            theme={theme}
-            title="My Wallet Address"
-            content={publicKey}
-            truncateContent
-            imageSize={50}
-            imageSource={avatar}
-            image={!avatar && unknownProfile}
-            copyButtonText="Copy address"
-            showCopyIcon={false}
-            onCopied={onPublicKeyCopied}
-          />
-          <BorderedBox
-            styles={styles}
-            theme={theme}
-            title="Fuse Network RPC Address"
-            content={web3ProviderUrl}
-            imageSize={50}
-            image={FuseLogo}
-            copyButtonText="Copy RPC"
-            showCopyIcon={false}
-          />
-        </View>
-      </Section>
+      <ScrollView contentContainerStyle={styles.contentContainerForBoxes} style={styles.containerForBoxes}>
```

Comment:
does that scroll view was defined in the design ?
if not - did you approved using it with @Esfir-E ?

Reply:
@serdiukov-o-nordwhale  New design of this screen extended by new box and new large title with subtitle and it impossible to place on the screen without scrollview. (even on the biggest mobile screen). I didn't approve using it with @Esfir-E  because I think it is obviously need it.

Reply:
@Zhigamovsky please discuss this change with @Esfir-E 
Probably she would suggest the other solution (e.g. adjust font / image / button sizes)

Reply:
@serdiukov-o-nordwhale Ok

Reply:
@Zhigamovsky please attach screen shots for UI tasks
@Esfir-E why cant we just add under the rpc another line "chainId: 122" why do we need another box? doesnt make sense

Reply:
@sirpy we have raised that concern in the issue [https://github.com/gooddollar/gooddapp/issues/2991](https://github.com/gooddollar/gooddapp/issues/2991)

Reply:
@serdiukov-o-nordwhale, Esfir approved scrolling.